src/modules/ie-objects/elasticsearch/elasticsearch.consts.ts

```diff
@@ -33,7 +33,9 @@ export enum SearchFilterField {
 	DESCRIPTION = 'description',
 	ERA = 'era',
 	LOCATION = 'location',
-	MAINTAINER = 'maintainer',
+	// TODO future: rename maintainer to maintainerId and maintainers to maintainerName and also change this in the client
```

Comment:
REQ: If we are adding it anyway can't we just add it already the correct way?

Reply:
no, this requires quite a lot of changes on the client, since this value is used all through the filtering mechanism

Reply:
REQ: add ticket for this in Jira